### PR TITLE
Jenkins documentation with manual build steps

### PR DESCRIPTION
**Pull request by samiyaakhtar**
Adding docs to setup Jenkins by manually adding build steps. The Jenkinsfile method yet to be added.

### CODE REVIEW

File: gitops/jenkins/ManifestGeneration.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+7. Click on `Build with Parameters` and hit `Build`:
+   ![](./images/build_with_parameters.png)
+8. Click on the build that just started, and view `Console Output` from left navigation to view the logs for this build. 
+   ![](./images/view_builds.png)
+   ![](./images/console_output.png)
+9. The build should clone the HLD repo, run the Fabrikate commands, and push the generated manifest files to your manifest repo. Head over to your manifest repository to see the changes that were pushed in!
+   ![](./images/test_from_jenkins.png)
+
+## Execute sudo commands from Jenkins build
```
Comment (andrebriggs) [inline gitops/jenkins/ManifestGeneration.md]: @samiyaakhtar what is the purpose of this section if we have already done a build with Jenkins. Is this the best practice in Jenkins?

File: gitops/jenkins/ManifestGeneration.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+4. Under the `Source code management` tab, paste the URL to the git repository. If you would like to build all branches, leave the box empty for `Branch specifier`. 
+   ![](./images/source_code_management.png)
+5. Under `Build Triggers`, select `GitHub hook trigger for GITScm polling`. 
+   ![](./images/build_triggers.png)
+6. Under `Build` click on `Add build step` -> `Execute shell` and paste the following code:
+   ```
+   #!/bin/bash
+
+    sudo -s
+    whoami
```
Comment (andrebriggs) [inline gitops/jenkins/ManifestGeneration.md]: @samiyaakhtar is this for debugging?

Comment (samiyaakhtar) [reply gitops/jenkins/ManifestGeneration.md]: Yes, my bad, will remove the debugging statements.

File: gitops/jenkins/ManifestGeneration.md (hunk 1 above)
Comment (samiyaakhtar) [reply gitops/jenkins/ManifestGeneration.md]: Yes, I will reformat this section a little more so it's clear that this is just a sub-step for the steps above.

**Pull request merged**